.travis.yml

```diff
@@ -86,6 +88,7 @@ matrix:
         - ./bin/vendor-libmysqlclient.sh ~/venv
         # Create the test database for `manage.py check --deploy`.
         - mysql -u root -e 'create database test_treeherder;'
+      --ignore=tests/jenkins/
```

Comment:
This needs to be moved to the `py.test` command line in the script section. You'll see there's already arguments such as `--ignore=tests/selenium/` so you just need to add another one for this path.